php/mixphp/composer.json

```diff
@@ -0,0 +1,10 @@
+{
+  "autoload": {
+    "psr-4": {
+      "App\\": "src/"
+    }
```

Comment:
just for my knowledge : is this not the default / standard in php packages ? I mean what if this is not defined explicitly ?

Reply:
This is a must https://getcomposer.org/doc/04-schema.md#psr-4